engine-rest/engine-rest/src/main/java/org/camunda/bpm/engine/rest/history/UserOperationLogRestService.java

```diff
@@ -48,4 +51,15 @@
   List<UserOperationLogEntryDto> queryUserOperationEntries(@Context UriInfo uriInfo,
                                                            @QueryParam("firstResult") Integer firstResult,
                                                            @QueryParam("maxResults") Integer maxResults);
+
+  @PUT
+  @Path("/{operationId}/set-annotation")
+  @Produces(MediaType.APPLICATION_JSON)
+  void setAnnotation(@PathParam("operationId") String operationId, AnnotationDto annotationDto);
+
+  @PUT
+  @Path("/{operationId}/clear-annotation")
+  @Produces(MediaType.APPLICATION_JSON)
+  void clearAnnotation(@PathParam("operationId") String operationId);
```

Comment:
let's return a `Response` object instead of `void`, so we're able to change what we return but stay backwards-compatible